scripts/run-screenshot-tests.sh

```
@@ -0,0 +1,441 @@
+        -noaudio \
+        -no-boot-anim \
+        -camera-back none \
+        -memory 3072 \
+        -partition-size 4096 &
+    
+    local emulator_pid=$!
+    print_status "Emulator started with PID: $emulator_pid"
+    
+    # Wait for emulator boot completion
+    print_status "Waiting for emulator to boot..."
+    adb wait-for-device shell 'while [[ -z $(getprop sys.boot_completed | tr -d '\r') ]]; do sleep 1; done'
```

Shellcheck (0.10.0)

[warning] \r is just literal 'r' here. For carriage return, use "$(printf '\r')" instead.

(SC1012)